login.go

```
@@ -32,6 +32,10 @@ func (c *LoginController) Authorize(ctx *app.AuthorizeLoginContext) error {
 
 // Generate runs the authorize action.
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
+	configuration, err := config.NewConfigurationData("config.yaml")
+	if err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
```

Why panic, when you can return an error?